mail/rspamd/Portfile

```diff
@@ -33,49 +30,80 @@ compiler.whitelist clang macports-clang-8.0 macports-clang-7.0 macports-clang-6.
 
 depends_build-append \
                     port:pkgconfig \
+                    port:ragel \
                     port:luajit \
-                    port:ragel
+                    port:gd2 \
+                    port:fann \
+                    port:libstemmer
 
 depends_lib-append  port:libevent \
                     port:libmagic \
                     port:sqlite3 \
                     port:openssl \
-                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
-                    path:lib/libopenblas.dylib:OpenBLAS \
                     port:gmime \
                     port:lua \
-                    port:pcre \
                     port:perl5 \
                     port:redis \
                     port:hiredis \
-                    port:zlib
+                    port:zlib \
+                    path:lib/pkgconfig/glib-2.0.pc:glib2 \
+                    path:lib/libopenblas.dylib:OpenBLAS
+
+# Critical dependency note: port:pcre and port:pcre2 break the rspamd binary;
+# use native /usr/lib/libpcre.dylib.  See https://github.com/rspamd/rspamd/issues/2884
+# Do not use this the library provided by port:pcre without pcre jit debugging
+# Do not use -DENABLE_PCRE2=ON without pcre jit debugging
+# # depends_build-append  port:pcre
+# # depends_build-append  port:pcre2
```

Comment:
Could you perhaps file a trac ticket against  MPs pcre port pointing at this issue. Just to document it incase anyone else stumbles on something similar or looks to addressing it ?

Reply:
https://trac.macports.org/ticket/58437

Reply:
Cheers